esp/src/src-react/components/Workunits.tsx

```diff
@@ -79,7 +79,7 @@ interface WorkunitsProps {
 }
 
 const emptyFilter: { [id: string]: any } = {};
-const defaultSort = { attribute: "Wuid", descending: true };
+export const defaultSort = { attribute: "Wuid", descending: true };
```

Comment:
Is exported needed?

Reply:
This wasn't being used because the router was always passing in something for the prop. It's getting used as part of the implicit import in the routes.